modules/context-utils/src/main/scala/trace4cats/context/syntax.scala

```diff
@@ -0,0 +1,15 @@
+package trace4cats.context
+
+object syntax {
```

Comment:
I think it might be good to include some `provide` syntax here too

Reply:
Done! See https://github.com/trace4cats/trace4cats/pull/911/commits/7b8fe6bf8def2551c73bd4df4642d9aae100b399